package.json

```diff
@@ -4,14 +4,14 @@
   "description": "cp-zen-platform",
   "main": "index.js",
   "scripts": {
+    "dev": "./node_modules/.bin/gulp dev",
```

Comment:
removing `&& nodemon` will break the development env

Reply:
k, will fix